corehq/apps/userreports/tests/test_app_manager_integration.py

```diff
@@ -62,12 +62,13 @@ def _get_column_property(column):
         default_case_property_datatypes = DEFAULT_CASE_PROPERTY_DATATYPES
         [row] = data_source.get_all_values(sample_doc)
         for result in row:
-            self.assertEqual(sample_doc[_get_column_property(result.column)], result.value)
-            if result.column.id in default_case_property_datatypes:
-                self.assertEqual(
-                    result.column.datatype,
-                    default_case_property_datatypes[result.column.id]
-                )
+            if result.column.id != "repeat_iteration":
```

Comment:
i didn't really have a proposal, was just noting. i think if we had a prefix (e.g. "system" or "ucr") where we could dump system columns and not allow any user generated columns to start with that it'd be nice long term. that prefix could just be an underscore. feel free to propose something if you want